tests/mo-timing-synchronization/OPS/package.opf

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package xmlns="http://www.idpf.org/2007/opf" xmlns:epub="http://www.idpf.org/2007/ops" version="3.0" xml:lang="en" unique-identifier="pub-id">
+  <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
+    <dc:coverage>Media Overlays</dc:coverage>
+    <dc:creator>Ivan Herman</dc:creator>
+    <dc:creator>Marisa DeMeglio</dc:creator>
+    <dc:date>2022-02-10</dc:date>
+    <dc:description>Tests whether the Reading System implements Media Overlays.</dc:description>
+    <dc:identifier id="pub-id">mo-timing-synchronization</dc:identifier>
```

Comment:
For consistency with other tests, consider making the abbreviation mol- instead of mo-.

Reply:
Yep, will come in next commit